src/Http/Controllers/Inertia/UserProfileController.php

```diff
@@ -21,6 +21,24 @@ public function show(Request $request)
     {
         return Jetstream::inertia()->render($request, 'Profile/Show', [
             'sessions' => $this->sessions($request)->all(),
+            'connectedAccounts' => $request->user()->connectedAccounts
+                ->map(function ($account) {
+                    return (object) [
+                        'id' => $account->id,
+                        'provider_name' => $account->provider_name,
+                        'created_at' => (new \DateTime($account->created_at))->format('d/m/Y H:i'),
+                    ];
+                }),
+            'hasPassword' => ! is_null($request->user()->password),
+            'socialiteProviders' => [
```

Comment:
I have seen this array repeated a few times through the PR. Isn't it better to move it to `src/Features` instead? Something like

```
public static function mapToEnabledSocialProviders(): array
{
    return  [
        'facebook' => Jetstream::hasSocialiteSupportFor('facebook'),
        'google' => Jetstream::hasSocialiteSupportFor('google'),
         'twitter' => Jetstream::hasSocialiteSupportFor('twitter'),
         'linkedin' => Jetstream::hasSocialiteSupportFor('linkedin'),
         'github' => Jetstream::hasSocialiteSupportFor('github'),
         'gitlab' => Jetstream::hasSocialiteSupportFor('gitlab'),
         'bitbucket' => Jetstream::hasSocialiteSupportFor('bitbucket'),
    ];
}
```

Then, it can simply be called as `Features::mapToEnabledSocialProviders()` from anywhere avoiding the repetition and having a single source of truth

Reply:
Yup, I'll add something like this in later today. Cheers!

Reply:
But,  if I only want to activate just one, how would I do it?

Reply:
You define it in your Jetstream config like so:

```php
'features' => [
    Jestream::socialite(['facebook' => true, 'github' => true]),
],
```

Any providers that are set to `true` will be displayed as options on your `login`, `register` routes as well as in the `user/profile` route.

You will need to define the tokens used in your `services.php` config in your Laravel project. 

An example project can be found [here](https://github.com/joelbutcher/jetstream-demo)

Reply:
Thats nice!! Thanks!!